webclient/static/js/submit_button.js

```diff
@@ -0,0 +1,9 @@
+$(document).ready(function () {
+  $('form.submit-spinner').on('submit', function() {
+    var submitButton = $(this).find('button[type=submit]');
```

Comment:
Já bych se používání `var` obecně úplně vyhnul a místo toho využíval `const` nebo `let` v případě, že budeme vyžadovat úpravu proměnné. Usnadní to hledání chyb v kódu.

Reply:
Ok